src/annotator/anchoring/text-range.ts

```diff
@@ -238,42 +236,38 @@ export class TextPosition {
  * of the range itself.
  */
 export class TextRange {
-  /**
-   * Construct an immutable `TextRange` from a `start` and `end` point.
-   *
-   * @param {TextPosition} start
-   * @param {TextPosition} end
-   */
-  constructor(start, end) {
+  public start: TextPosition;
+  public end: TextPosition;
+
+  constructor(start: TextPosition, end: TextPosition) {
     this.start = start;
     this.end = end;
   }
 
   /**
-   * Return a copy of this range with start and end positions relative to a
-   * given ancestor. See `TextPosition.relativeTo`.
+   * Create a new TextRange relative to `element`.
```

Comment:
This first sentence ("Create a new TextRange") is redundant given the second one ("Return a new TextRange").